### PR TITLE
[Refactor] Private constructor for singleton

### PR DESCRIPTION
**Pull request by e0c9**
## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [x] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have create an issue on (Fix #ISSUE), and have described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/service/ExecuteEnv.java [hunk 1]
```diff
@@ -42,11 +43,6 @@ public static ExecuteEnv getInstance() {
         return INSTANCE;
     }
 
-    public static void setup() {
-        ExecuteEnv env = getInstance();
-        env.scheduler = new ConnectScheduler(Config.qe_max_connection);
-    }
-
     public ConnectScheduler getScheduler() {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/service/ExecuteEnv.java:46]: If you do this, then the `getScheduler()` and `getMultiLoadMgr()` need to be changed too.

**Review by morningman — CHANGES_REQUESTED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/service/ExecuteEnv.java (hunk 1 above)
Comment (e0c9) [reply fe/fe-core/src/main/java/org/apache/doris/service/ExecuteEnv.java:46]: I think this change will not affect these two methods. Can you explain it？

Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/service/ExecuteEnv.java:46]: Sorry for replying late.
I misunderstood. it looks good to me.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**